core/templates/pages/teach-page/teach-page.component.html

```diff
@@ -616,7 +616,7 @@ <h2 [innerHTML]="'I18N_TEACH_PAGE_SIX_TITLE' | translate"></h2>
   }
 
   .teach-page .teach-with-oppia-image img {
-    border: 4px solid #FFFFFF;
+    border: 4px solid #FFF;
```

Comment:
Should this be in lowercase as used in other places?

Reply:
@jameesjohn Thank you very much for the suggestion!
I think there are multiple instances where uppercase is used as the ideal style in not mention in the [wiki](https://github.com/oppia/oppia/wiki/Coding-style-guide#css). @DubeySandeep What are your thoughts upon this, should we add this convention in wiki? 
And then after that we can write a lint check for it as well.

Reply:
@DubeySandeep ping.

Reply:
Let's have a consistent pattern everywhere. Also, it's expected to have lowercase in all the places. ([ref](https://google.github.io/styleguide/htmlcssguide.html#Capitalization)).

I'll add this in the wiki page as well.

Reply:
Would you like to add [color-hex-case](https://stylelint.io/user-guide/rules/color-hex-case) in your next PR?

Reply:
I'm going to merge this PR as it has changes in 60+ files and is prone to merge conflicts!

Can you please raise a PR with the color-hex-case check?

Reply:
@DubeySandeep If you are asking me ? then I would love to do it.